src/Plugin/TwigPlugin/GetUniqueId.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\jumpstart_ui\Plugin\TwigPlugin;
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Extend Drupal's Twig_Extension class
+ */
+class GetUniqueId extends \Twig_Extension {
```

Comment:
Can we make this generic. this class could contain several twig functions/filters

Reply:
Hmm. Yeah, makes sense.